windows/CodePushInvalidUpdateException.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+
+namespace CodePush.ReactNative
+{
+    class CodePushInvalidUpdateException : Exception
```

Comment:
Actually I think we can replace this custom exception type and just throw [InvalidOoerationException](https://msdn.microsoft.com/en-us/library/system.invalidoperationexception(v=vs.110).aspx) from the download method.